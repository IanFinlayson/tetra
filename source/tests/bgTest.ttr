def main():
   x = 0
   print("Just set x = 0 and x = ", x, "\n")
   background:
      x += 10
   #delaying here virtually assures that the background thread executes
   for q in [1 ... 1000]:
      pass

   print("At the end of main an x = ", x, "\n")
