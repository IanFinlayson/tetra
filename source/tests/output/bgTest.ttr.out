Just set x = 0 and x = 0
At the end of main an x = 10

